docs/service-introspection-onboarding.md

```diff
@@ -149,32 +149,59 @@ by service introspection.
 
 The CD release pipeline updates the docker image number in the HLD.
 
-If you are not using multi-stage pipelines, paste the following task towards the end of your release step in the release
-pipeline:
+The release pipeline can be setup in two different ways: There are two options
+to setup the ACR to HLD step.
+
+****Option 1:**** As an
+[Azure Release Pipeline](https://docs.microsoft.com/en-us/azure/devops/pipelines/release/?view=azure-devops)
+
+**Instructions:**
+
+Paste the following task towards the end of your release step in the release
+pipeline in the Azure DevOps portal:
 
 ```yaml
-latest_commit=$(git rev-parse --short HEAD)
-VERSION_TO_DOWNLOAD=$(curl -s "https://api.github.com/repos/CatalystCode/spk/releases/latest" | grep "tag_name" | sed -E 's/.*"([^"]+)".*/\1/') 
-echo "Downloading SPK version $VERSION_TO_DOWNLOAD" && wget "https://github.com/CatalystCode/spk/releases/download/$VERSION_TO_DOWNLOAD/spk-linux"
-chmod +x ./spk-linux
-./spk-linux deployment create  -n $(ACCOUNT_NAME) -k $(ACCOUNT_KEY) -t $(TABLE_NAME) -p $(PARTITION_KEY)  --p2 $(Release.ReleaseId) --hld-commit-id $latest_commit --env $(Release.EnvironmentName) --image-tag $(Build.BuildId)
+latest_commit=$(git rev-parse --short HEAD) VERSION_TO_DOWNLOAD=$(curl -s
+"https://api.github.com/repos/CatalystCode/spk/releases/latest" | grep
+"tag_name" | sed -E 's/.*"([^"]+)".*/\1/') echo "Downloading SPK version
+$VERSION_TO_DOWNLOAD" && wget
+"https://github.com/CatalystCode/spk/releases/download/$VERSION_TO_DOWNLOAD/spk-linux"
+chmod +x ./spk-linux ./spk-linux deployment create  -n $(ACCOUNT_NAME) -k
+$(ACCOUNT_KEY) -t $(TABLE_NAME) -p $(PARTITION_KEY)  --p2 $(Release.ReleaseId)
+--hld-commit-id $latest_commit --env $(Release.EnvironmentName) --image-tag
+$(Build.BuildId)
```

Comment:
As far as I can see, all this diff is doing is adding extra lines between the yaml, which will fail the bash task. You might want to fix the new lines. Other than the new lines, i don't see a diff here

Reply:
@edaena I think this is still valid, may want to remove this diff altogether